### ISSUE TITLE
BSP issue with nRF Connect SDK v2.0.0

### ISSUE THREAD

**Issue by jpwright** [u-blox/u-blox-sho-OpenCPU#8]
Building the `hello_world` Zephyr sample for the `ubx_evknorab10_nrf5340_cpuapp` board gives me the following error using nRF Connect SDK v2.0.0:

```
> Executing task: nRF Connect: Generate config ubx_evknorab10_nrf5340_cpuapp for /home/jason/hello_world <

Building hello_world
west build --build-dir /home/jason/hello_world/build /home/jason/hello_world --pristine --board ubx_evknorab10_nrf5340_cpuapp -- -DNCS_TOOLCHAIN_VERSION:STRING="NONE"

-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/jason/hello_world
-- Found Python3: /usr/bin/python3.8 (found suitable exact version "3.8.10") found components: Interpreter 
-- Cache files will be written to: /home/jason/.cache/zephyr
-- Zephyr version: 3.0.99 (/home/jason/ncs-2.0.0/zephyr)
-- Found west (found suitable version "0.12.0", minimum required is "0.7.1")
-- Board: ubx_evknorab10_nrf5340_cpuapp
-- Found dtc: /usr/bin/dtc (found suitable version "1.5.0", minimum required is "1.4.6")
-- Found toolchain: gnuarmemb (/home/jason/gnuarmemb)
-- Found BOARD.dts: /home/jason/ncs-2.0.0/zephyr/boards/arm/ubx_evknorab10_nrf5340/ubx_evknorab10_nrf5340_cpuapp.dts
devicetree error: missing data after phandle in <Property 'pwms' at '/pwmleds/pwm_led_0' in '/home/jason/ncs-2.0.0/zephyr/misc/empty_file.c'>
CMake Error at /home/jason/ncs-2.0.0/zephyr/cmake/modules/dts.cmake:213 (message):
  gen_defines.py failed with return code: 1
Call Stack (most recent call first):
  /home/jason/ncs-2.0.0/zephyr/cmake/modules/zephyr_default.cmake:121 (include)
  /home/jason/ncs-2.0.0/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:51 (include)
  /home/jason/ncs-2.0.0/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:76 (include_boilerplate)
  /home/jason/hello_world/build/CMakeLists.txt:5 (find_package)


-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/bin/cmake -DWEST_PYTHON=/usr/bin/python3 -B/home/jason/hello_world/build -S/home/jason/hello_world -GNinja -DBOARD=ubx_evknorab10_nrf5340_cpuapp -DNCS_TOOLCHAIN_VERSION:STRING=NONE
The terminal process terminated with exit code: 1.

Terminal will be reused by tasks, press any key to close it.
```

**Comment by ghost** [comment]
We are working on updating these files to be compatible with NCS v2.0.0. They were last released around the time of v1.6.1.

A colleague indicated this particular issue can be fixed with :
Change line 41 of ubx_evknorab10_nrf5340_cpuapp.dts to:
pwms = <&pwm0 28 PWM_MSEC(20) PWM_POLARITY_INVERTED>;

**Comment by ghost** [comment]
ubx_evknorab10_nrf5340 has been updated based on nrf5340dk_nrf5340 for NCS v2.0.2 (identical files in v2.0.0 and v2.0.1). Samples built with NCS v2.0.x appear to work now with the new file set.
